package.json

```
@@ -12,18 +12,17 @@
 		"setup": "npm ci && npm run export-version && npm run build --production && npm run commands:deploy",
 		"commands:deploy": "node . --deploy",
 		"commands:revoke": "node . --revoke",
-		"release": "./node_modules/.bin/ts-node -P tsconfig.test.json ./scripts/release.ts",
+		"release": "./node_modules/.bin/ts-node ./scripts/release.ts",
 		"db:init": "./node_modules/.bin/prisma migrate resolve --applied 20220915022732_initial_state  # TODO: Use this to set initial tables on the database",
 		"db:migrate": "./node_modules/.bin/prisma migrate deploy",
 		"db:introspect": "./node_modules/.bin/prisma db pull",
 		"db:generate": "./node_modules/.bin/prisma generate --schema ./prisma/schema.prisma",
 		"db:migrations:generate": "./node_modules/.bin/prisma migrate dev --create-only",
-		"lint": "./node_modules/.bin/eslint . --ext .ts",
+		"lint": "./node_modules/.bin/eslint .",
 		"lint:fix": "npm run lint -- --fix",
-		"build": "rm -rf dist && ./node_modules/.bin/tsc -p tsconfig.prod.json",
+		"build": "rm -rf dist && ./node_modules/.bin/tsc",
 		"export-version": "./node_modules/.bin/genversion ./src/constants/version.ts -es",
-		"test": "./node_modules/.bin/jest",
-		"test:watch": "npm run test -- --watch",
+		"test": "./node_modules/.bin/vitest",
```

I really like watch mode:

```suggestion
		"test": "./node_modules/.bin/vitest run",
		"test:watch": "./node_modules/.bin/vitest watch",
```

---

Watch mode is the default when running vitest outside a CI/CD pipeline. It switches automatically - a "smart default". This is super useful because it means we can have a single `test` command to be used in local development and our GitHub pipelines. See [here](https://vitest.dev/guide/features.html#watch-mode) for more details.

---

Alrighty then 👍